src/core.js

```diff
@@ -31,7 +31,7 @@ var globalStartCalled = false;
 var runStarted = false;
 
 // Figure out if we're running the tests from a server or not
-QUnit.isLocal = !( defined.document && window.location.protocol !== "file:" );
+QUnit.isLocal = ( window && window.location && window.location.protocol === "file:" );
```

Comment:
If we update our babel setup we could use 🤔 

`window?.location?.protocol === 'file:'`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining